### PR TITLE
Add view model

### PR DESCRIPTION
**Pull request by HamiiidAsgarian**
Standard features are completed and the main feature is the account page. I will fix it when project-specific needs are decided.

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -67,5 +68,7 @@ flutter {
 }
 
 dependencies {
+    // implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```
Comment (Codologyst) [inline android/app/build.gradle:71]: clean this up.

File: lib/view/widgets/my_rounded_button.dart [hunk 2]
```diff
@@ -2,9 +2,9 @@ import 'package:flutter/material.dart';
 
 import '../../consts.dart';
 
-class MyRoundButton extends StatefulWidget {
+class MyRoundButton extends StatelessWidget {
```
Comment (Codologyst) [inline lib/view/widgets/my_rounded_button.dart:5]: make this to all the widgets if you can.

**Review by Codologyst — APPROVED** [review]
This should do until we handle the real project and backed. make sure you are keeping it clean and fresh.

**Comment by HamiiidAsgarian** [comment]
Most of the UI is done and the project will continue when the backend is ready.

**Pull request merged**